content/blog/2019/11/2019-11-22-welcome-to-the-matrix.adoc

```diff
@@ -0,0 +1,394 @@
+---
+layout: post
+title: "Welcome to the Matrix"
+tags:
+- pipeline
+- plugins
+- declarative
+- matrix 
```

Comment:
Social media image would be recommended, especially since the post title is not self-descriptive